style.css

```
@@ -1 +1,246 @@
 /*  Add your styling here */
+
+nav {
+    font-family: 'Lobster', cursive;
+    font-size: 20px;
+}
+
+.topnav {
+    overflow: hidden;
+    background-color: rgb(33, 248, 248);
+    position: relative;
+  }
+  
+.topnav #myLinks {
+    display: none;
+}
+  
+.topnav a {
+    float: left;
+    color: black;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 17px;
+}
+  
+.topnav a.icon {
+    float: right;
+}
+  
+.topnav a:hover {
+    background-color: rgb(255, 149, 193);
+    color: white;
+    border:rgb(255, 149, 193);
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+
+header {
+    background-color: rgb(33, 248, 248);
+    padding-bottom: 40px;
+    display: flex;
+}
+
+h1 {
+    align-self: flex-end;
+    text-align: right;
+    flex-grow: 2;
+    margin-right: 50px;
+    font-family: 'Lobster', cursive;
+    font-size: 14px;
+}
+
+.logo {
+    width: 150px;
+    height: 150px;
+}
+
+.intro {
+    background-color: rgb(243, 236, 141);
+    text-align: justify;
+    font-size: 14px;
+}
+
+h2, p {
+    font-family: 'Lobster', cursive;
+    margin-left: 40px;
+    padding: 40px 40px 40px 10px;
+}
+
+p {
+    padding-top: 20px;
+}
+
+h2 {
+    padding-bottom: 10px;
+}
+
+.giantCupcake {
+    width: 200px;
+    height: 250px;
+    border: 2px solid rgb(243, 236, 141);
+}
+
+.cakes {
+    width: 100px;
+    height: 75px;
+    margin: 2px;
+    border: 2px solid rgb(255, 149, 193);
+}
+
+.giantCake, .cupcakes {
+    display: flex;
+    align-items: center;
```

```suggestion
    align-items: center;
```
What happens if you comment this line?